Modules/Setting/Resources/views/admin/fields/plain/textarea.blade.php

```diff
@@ -1,8 +1,9 @@
+<?php $defaultValue = isset($moduleInfo['default']) ? $moduleInfo['default']: ''; ?>
 <div class='form-group'>
     {!! Form::label($settingName, trans($moduleInfo['description'])) !!}
     <?php if (isset($dbSettings[$settingName]) && $dbSettings[$settingName]->plainValue !== null): ?>
-        {!! Form::textarea($settingName, old($settingName, $dbSettings[$settingName]->plainValue), ['class' => 'form-control', 'placeholder' => trans($moduleInfo['description'])]) !!}
+        {!! Form::textarea($settingName, old($settingName, $dbSettings[$settingName]->plainValue) ?: $defaultValue, ['class' => 'form-control', 'placeholder' => trans($moduleInfo['description'])]) !!}
```

Comment:
Looks like you forget the ternary here 😄

Reply:
No, I do not removed it because the `$dbSettings[$settingName]->plainValue` could be different from the `$defaultValue `.  Do you think it can never happen?

Reply:
Oh yes I see, correct.